### ISSUE TITLE
Share image / sharing on the mobile version of arquivo.pt

### ISSUE THREAD

**Issue by Fernando-Melo** [arquivo/pwa-technologies#586]
When the image viewer is opened, if we attempt to share (copy link) it will copy the image link, when it should somehow share the image Card (the viewer with the image and the image details).

One other suggestion by Ivo was to use the Web share API https://w3c.github.io/web-share/ , although it currently only works in Chrome mobile and Safari, and it needs a custom fallback for the remaining browsers.

**Comment by igobranco** [comment]
This could only be done if we create a new screen that displays the image card. This page is what could be shared.

Another thing is on image search screen, on the image viewer, to update browser history with current URL + the current image md5 checksum. So if ther user refresh the image search it opens again the current modal. If it shares that page it shares the arquivo.pt image search and that image opened.

**Comment by igobranco** [comment]
The share option have been removed. Currently we only have the copy link feature.

Why the user wants to copy the link? To share it.

I've added the Web share API call to the copy link action. 
Currently it's supported on chrome on mobile and safari on iOS. 

What happens on chrome on android:
![Screenshot_2020-02-14-15-51-16-053_android](https://user-images.githubusercontent.com/67018/74547492-18c2a000-4f44-11ea-94f7-0695d2a321bf.jpg)

On desktop the copy link didn't change.
